perf/src/lib.rs

```diff
@@ -32,6 +71,36 @@ impl CongestionAlgorithm {
     }
 }
 
+pub fn build_transport_config(
+    common: &CommonOpt,
+    #[cfg(feature = "qlog")] name: &str,
+) -> io::Result<TransportConfig> {
+    let mut transport = quinn::TransportConfig::default();
+    transport.initial_mtu(common.initial_mtu);
+
+    if let Some(initial_rtt) = common.initial_rtt {
+        transport.initial_rtt(Duration::from_millis(initial_rtt));
+    }
+
+    if common.ack_frequency {
+        transport.ack_frequency_config(Some(quinn::AckFrequencyConfig::default()));
+    }
+
+    if let Some(cong_alg) = common.cong_alg {
+        transport.congestion_controller_factory(cong_alg.build());
+    }
+
+    #[cfg(feature = "qlog")]
+    if let Some(qlog_file) = &common.qlog_file {
+        let mut qlog = quinn::QlogConfig::default();
+        qlog.writer(Box::new(std::fs::File::create(qlog_file)?))
```

Comment:
Nit: please import `File` as before, and also all the other symbols here (`TransportConfig`, `AckFrequencyConfig`, `QlogConfig`, etc).